### PR TITLE
Update-821: Dev rack and misc docs-2

### PR DESCRIPTION
**Pull request by ntner**
### What is the feature/fix?

- Rewrote development rack docs for Ubuntu and macOS Intel/x86
- Added macOS-M1/ARM64 page
- Updated various misc docs errors and omission.

- Proofreading Fixes

### Does it has a breaking change?

No, just docs updates

### How to use/test it?

Read the docs and test

### Checklist
- [ ] New coverage tests
- [ ] Unit tests passing
- [ ] E2E tests passing
- [ ] E2E downgrade/update test passing
- [ ] Documentation updated
- [ ] No warnings or errors on Deepsource/Codecov

### CODE REVIEW

File: docs/installation/development-rack/macos-m1.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+For this example the command would be:
+`minikube start --kubernetes-version=v1.23.0 --insecure-registry="https://registry.test-local.macdev.convox.cloud" --driver=docker` 
+- If you receive an error regarding docker permissions, you can run the group command given in the output. 
+
+Now enable Minikube ingress and ingress-dns addons: 
+` minikube addons enable ingress` 
+` minikube addons enable ingress-dns` 
+  
+
+Then install the local development rack with the command `convox rack install local <RACK_NAME> -v <VERSION>`
```
Comment (nightfury1204) [inline docs/installation/development-rack/macos-m1.md:41]: here also os=mac
```
convox rack install local <RACK_NAME> -v <VERSION> os=mac
```

File: docs/installation/development-rack/macos.md [hunk 2]
```diff
@@ -4,68 +4,56 @@ draft: false
...
-    $ sudo mv /tmp/convox /usr/local/bin/convox
-    $ sudo chmod 755 /usr/local/bin/convox
-```
+Now enable Minikube ingress and ingress-dns addons: 
+` minikube addons enable ingress` 
+` minikube addons enable ingress-dns`
+  
 
-## Installation
+Then install the local development rack with the command `convox rack install local <RACK_NAME> -v <VERSION>`
```
Comment (nightfury1204) [inline docs/installation/development-rack/macos.md:41]: here also os=mac

`convox rack install local <RACK_NAME> -v <VERSION> os=mac`

**Review by Twsouza — APPROVED** [review]

File: docs/installation/development-rack/macos.md (hunk 2 above)
Comment (ntner) [reply docs/installation/development-rack/macos.md]: fixed

File: docs/installation/development-rack/macos-m1.md (hunk 1 above)
Comment (ntner) [reply docs/installation/development-rack/macos-m1.md]: fixed

**Review by nightfury1204 — COMMENTED** [review]
:fire:

**Review by Twsouza — APPROVED** [review]

**Review by nightfury1204 — DISMISSED** [review]

**Review by nightfury1204 — APPROVED** [review]

**Review by Twsouza — APPROVED** [review]

**Pull request merged**